Product/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+
+class MainCategory(models.Model):
+	name  = models.CharField(max_length=45)
+
+	class Meta:
+	    db_table = 'main_categories'
+
+class Category(models.Model):
+	name          = models.CharField(max_length = 45)
+	main_category = models.ForeignKey(MainCategory, on_delete=models.CASCADE)
+
+	class Meta:
+	    db_table = 'categories'
+
+class SubCategory(models.Model):
+	name          = models.CharField(max_length = 45)
+	category      = models.ForeignKey(Category, on_delete =models.CASCADE)
+
+	class Meta:
+	    db_table = 'sub_categories'
+
+
+class ProductGroup(models.Model):
+	name  = models.CharField(max_length = 45, null=True)
+
+	class Meta:
+	    db_table = 'product_group'
+
+class Products(models.Model):	
+	name            = models.CharField(max_length = 45)
+	price 		= models.IntegerField(default = 0)	
+	point		= models.IntegerField(default = 0)
+	thumbnail       = models.CharField(max_length = 100)
+	description     = models.TextField()
+	info		= models.TextField()
+	notice          = models.TextField()
+	sale_price	= models.IntegerField(null=True)
+	sub_category    = models.ForeignKey(SubCategory, on_delete=models.CASCADE)
+	hover_image     = models.CharField(max_length=100, null = True)
+	product_group	= models.ForeignKey(ProductGroup, on_delete = models.CASCADE)
+	sub_text	= models.CharField(max_length=45, null=True)
+
+
+	class Meta:
+	    db_table = 'products'
+
+
+class DetailImageUrl(models.Model):
+        name         = models.ForeignKey(Products, on_delete=models.CASCADE)
```

Comment:
```suggestion
        product         = models.ForeignKey(Products, on_delete=models.CASCADE)
```
이 컬럼은 디테일 이미지의 이름이 아니라 제품에 연결되어 있기 때문에, product로 바꾸시는 것이 좋습니다.